supplemental_notebooks/2021-07-01_muddiest_points.Rmd

```diff
@@ -0,0 +1,269 @@
+---
+title: "Day 4: Intro to pathway analysis supplemental notebook"
+author: Jaclyn Taroni
+date: 2021-07-01
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+## Introduction
+
+This notebook addresses some questions that came up during or after the fourth day of the CCDL workshop.
+
+- [Saving GSEA plots for all significant pathways](#saving-gsea-plots-for-all-significant-results)
+- [GSEA significance](#a-little-more-on-GSEA-significance)
+
+### How to use this notebook: 
+
+While it is fine to just read this page as is, if you want to get the Rmarkdown notebook itself to work with, you can follow the following steps:
+
+1. Click the `Code` button in the upper right of this page and choose `Download Rmd` to save this notebook (`2021-07-01_muddiest_points.Rmd`) to your computer.
+If you have RStudio installed on your computer, you can then open the file directly.
+
+Otherwise you can add it to the CCDL RStudio server to work with it there with the following additional steps:
+
+2. Navigate to RStudio server.  
+3. In the `File` pane, navigate to `training-modules/scRNA-seq`.     
+4. Click the `Upload` button in the `File` pane.   
+5. Click `Choose file` and find this `Rmd` file on your computer, then click `OK`  
+
+Now you can open up this file and run it on the Server. 
+
+## Libraries
+
+```{r}
+library(magrittr)
+# Package to run GSEA
+library(clusterProfiler)
+# Package that contains the MSigDB gene sets in tidy format
+library(msigdbr)
+```
+
```

Comment:
Adding a chunk to install ggridges
~~~suggestion

For some later plots, we are going to use the `ggridges` package, which may not be installed, so let's install it here (only if it is not already installed):

```{r}
if (! "ggridges" %in% installed.packages()){
  install.package("ggridges")
}
```
~~~

Reply:
I guess I must have installed it myself previously!